pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -778,7 +778,7 @@ assertNoAdditions {
     };
 
   codewindow-nvim = super.codewindow-nvim.overrideAttrs {
-    dependencies = [ self.nvim-treesitter ];
+    dependencies = [ self.nvim-treesitter-legacy ];
```

Comment:
Do all of the plugins fail to build/run with the new plugin ? Or are you just assuming they aren't compatible?